src/components/SingleStoryCard/index.css

```diff
@@ -0,0 +1,46 @@
+:root {
+  --title-browse-color: #6c567b;
+  --storyBody-color: #000000;
+  --background-color: #fc9e01;
+  --hover-browse-color: #074081;
+  --shadow-color: rgba(0, 0, 0, 0.25);
+}
+
+.articleContainer {
+  margin: 3rem auto 5rem auto;
+  text-align: center;
+}
+
+.articleTitle {
+  color: var(--title-browse-color);
+}
+
+.articleBody {
+  line-height: 2rem;
+  margin: 3rem auto;
+  color: var(--storyBody-color);
+  width: 80%;
+}
+
+.readMoreButton {
+  color: #fff;
```

Comment:
this should be a variable or just white instead of the # code so it is more readable

Reply:
edited